src/index.html

```diff
@@ -4,9 +4,342 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="logo"
+        class="header__logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
```

Comment:
In the all next projects better name this element `item`.

In this task could leave it without changes.
We don't use non semantic tags in class names(such as p, ul, li, div) but could use semantic tags as a class names(header, nav, footer).

```suggestion
          <li class="nav__item">
```